x/upgrade/types/plan_test.go

```diff
@@ -58,41 +57,12 @@ func TestPlanValid(t *testing.T) {
 		p     types.Plan
 		valid bool
 	}{
-		"proper by height": {
-			p: types.Plan{
-				Name:   "all-good",
-				Height: 123450000,
-			},
-			valid: true,
-		},
-		"no name": {
-			p: types.Plan{
-				Height: 123450000,
-			},
-		},
-		"time-base upgrade": {
-			p: types.Plan{
-				Time: time.Now(),
-			},
-		},
-		"IBC upgrade": {
-			p: types.Plan{
-				Height:              123450000,
-				UpgradedClientState: &codectypes.Any{},
-			},
-		},
 		"no due at": {
 			p: types.Plan{
 				Name: "missing",
 				Info: "important",
 			},
 		},
-		"negative height": {
-			p: types.Plan{
-				Name:   "minus",
-				Height: -12345,
-			},
-		},
```

Comment:
why do you remove test cases related to name, Height, IBC?

Reply:
I reverted it. thanks